test/test.js

```diff
@@ -1,13 +1,26 @@
+"use strict";
```

Comment:
this one https://travis-ci.org/ewnd9/anki-apkg-export/jobs/179224350#L387

I can downgrade / search another package. But suppose that  the game isn't worth the candle